Form-Controls/index.html

```diff
@@ -1,28 +1,104 @@
+    </svg>
+    <h1>Choose your Mondrian Art T-shirt</h1>
   </header>
-  <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <input type="radio" id="com-a" name="mondrian-group" value="com-a" required />
+    <input type="radio" id="com-c" name="mondrian-group" value="con-c" />
+    <input type="radio" id="com-red-blue-yel" name="mondrian-group" value="com-red-blue-yel" />
+
+    <main>
+      <div class="col-1">
+        <label for="fname">First Name <span class="required">*</span></label>
+        <input type="text" id="fname" name="fname" placeholder="First Name" required minlength="2" />
+      </div>
+      <div class="col-1">
+        <label for="lname">Last Name <span class="required">*</span></label>
+        <input type="text" id="lname" name="lname" placeholder="Last Name" required minlength="2" />
+      </div>
+      <div class="col-2">
+        <label for="email">Email Address <span class="required">*</span></label>
+        <input type="text" id="email" name="email" placeholder="Email" pattern="[^ @]*@[^ @]*" required />
+      </div>
+
+      <div class="col-2">
+        <fieldset>
```

Comment:
I love this! Excellent use of semantic HTML and a lovely looking, creative input for the designs!